snapcraft.yaml

```diff
@@ -0,0 +1,36 @@
+name: dvc
+summary: Data Version Control
+description: Git for Data & Models https://dvc.org
+adopt-info: dvc
+grade: stable
+confinement: classic  # for same reasons as snap `git-ubuntu`
+base: core18
+license: Apache-2.0
+parts:
+  dvc:
+    plugin: python
+    python-packages: [wheel]
+    source: .
+    build-packages:
+    - git
+    - libc-dev
+    stage-packages:
+    - git
+    override-pull: |
+        snapcraftctl pull
+        echo 'PKG = "snap"' > $SNAPCRAFT_PART_SRC/dvc/utils/build.py
```

Comment:
Need to add an entry to https://github.com/iterative/dvc/blob/master/dvc/updater.py#L124 for snap. I do understand that it updates it automatically, but still.

Reply:
@efiop see https://github.com/iterative/dvc/blob/ba12e1c4571a113c44b7756b67c453655ea80595/snapcraft.yaml#L34 - this just skips checking for updates

Reply:
also `snap` packages automatically update 4 times per day

Reply:
Not sure I get to point. So you've hacked it to not check for updates (why?) and we don't have an update notifications too.

Reply:
Having update checks and notifications is contrary to one of the purposes of snap packages. For example, the firefox snap also has both update checks and ability to perform updates disabled.
`snap` itself is meant to handle this 4 times a day; including automatically rolling back if an update fails. Displaying an "out of date message" is equivalent to saying "please wait up to 6 hours for an update to automatically get downloaded or force it to happen now by running `snap install` again or pin the version using `snap` to disable updates"

Reply:
Ah, got it. I didn't get right away that CI was for dvc and not for snap. But we still need to add an entry to updater.py, as it might break. Let's add a safeguard there, no real harm in that.

Reply:
yes `CI` is an env var only currently checked in `updater.py` (and I think I made a note somewhere that it may break in future if things change)

Reply:
hope 40d1e29d36a60b5348d0dcd342680db55276d5ed addresses this

Reply:
just realised we should just check `PKG` instead. Presumably that was the entire point of that var.